code/modules/mob/living/living_defense.dm

```diff
@@ -50,6 +50,8 @@
 
 
 /mob/living/bullet_act(var/obj/item/projectile/P, var/def_zone)
+	if (P.is_hot() >= HEAT_MOBIGNITE_THRESHOLD)
```

Comment:
do projectiles work ? IIRC you said they didn't.

Can you ignite someone by dousing them in flame and firing a laser at them?

Reply:
They work now and yes you can.